tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -106,66 +110,86 @@
+                    <TextBlock x:Uid="ProviderTextBlock" VerticalAlignment="Center" Margin="50 0 5 0"/>
+                    <ComboBox x:Uid="ProviderSelectionComboBox" x:Name="ProviderSelectionComboBox" Margin="0 3 5 0"
+                          ItemsSource="{x:Bind ViewModel.Providers}" SelectedIndex="{x:Bind ViewModel.SelectedProviderIndex, Mode=TwoWay}">
+                        <i:Interaction.Behaviors>
+                            <ic:EventTriggerBehavior EventName="SelectionChanged">
+                                <ic:InvokeCommandAction Command="{x:Bind ViewModel.ProviderHandlerCommand}" />
+                            </ic:EventTriggerBehavior>
+                        </i:Interaction.Behaviors>
+                    </ComboBox>
+                    <!-- Sort field -->
+                    <TextBlock x:Uid="SortByTextBlock" VerticalAlignment="Center" Margin="10 0 5 0"/>
+                    <ComboBox x:Uid="SortSelectionComboBox" x:Name="SortSelectionComboBox" Margin="0 3 13 0"
+                          SelectedIndex="{x:Bind ViewModel.SelectedSortIndex, Mode=TwoWay}">
+                        <ComboBoxItem x:Uid="SortByName"/>
+                        <ComboBoxItem x:Uid="SortByNameDesc"/>
+                        <ComboBoxItem x:Uid="SortByAltName"/>
+                        <ComboBoxItem x:Uid="SortByAltNameDesc"/>
+                        <ComboBoxItem x:Uid="SortByLastConnected"/>
+                        <i:Interaction.Behaviors>
+                            <ic:EventTriggerBehavior EventName="SelectionChanged">
+                                <ic:InvokeCommandAction Command="{x:Bind ViewModel.SortHandlerCommand}"/>
+                            </ic:EventTriggerBehavior>
+                        </i:Interaction.Behaviors>
+                    </ComboBox>
+                    <!--  'Sync' button -->
+                    <Button Padding="40 5 30 8" Command="{x:Bind ViewModel.SyncButtonCommand}" Margin="0 3 13 0">
+                        <StackPanel Orientation="Horizontal">
+                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE72C;"/>
+                            <TextBlock x:Uid="SyncButtonTextBlock" Margin="8 0 0 0"/>
+                        </StackPanel>
+                    </Button>
+                </StackPanel>
+        </Grid>
+
+        <!-- Last synced text block-->
+        <StackPanel Grid.Row="2" Orientation="Horizontal" HorizontalAlignment="Right" MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
+            <TextBlock x:Uid="LastSyncTextBlock"
+                       VerticalAlignment="Center"
+                       Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                       Margin="0 12 3 12" />
+            <TextBlock Text="{x:Bind ViewModel.LastSyncTime, Mode=OneWay}"
+                       VerticalAlignment="Center" Foreground="{ThemeResource TextFillColorSecondaryBrush}"/>
         </StackPanel>

```

Comment:
I got this one slightly wrong -- the StackPanel sees HorizontalAlignment=Right and wants to stick to the right, no matter the available size. What you have right now gives you this when the window is wide:
![image](https://github.com/microsoft/devhome/assets/47155823/c38491dc-ecf6-4d81-af1f-39727ffd3e3c)

Here's the fix I would use, I tried it out to make sure it works:
```suggestion
        <!-- Last synced text block-->
        <Grid Grid.Row="2" MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
            <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" >
                <TextBlock x:Uid="LastSyncTextBlock"
                           VerticalAlignment="Center"
                           Foreground="{ThemeResource TextFillColorSecondaryBrush}"
                           Margin="0 12 3 12" />
                <TextBlock Text="{x:Bind ViewModel.LastSyncTime, Mode=OneWay}"
                           VerticalAlignment="Center"
                           Foreground="{ThemeResource TextFillColorSecondaryBrush}"/>
            </StackPanel>
        </Grid>
```